doc/source/drivers/vector/xodr.rst

```diff
@@ -0,0 +1,198 @@
+
+-  .. oo:: DISSOLVE_TIN
+      :choices: YES, NO
+      :default: NO
+
+      Whether to dissolve triangulated surfaces. By setting this option to YES, the TIN layers *Lane* and *RoadMark* of geometry type :cpp:class:`OGRTriangulatedSurface` will be simplified to single, simple :cpp:class:`OGRPolygon` geometries. This performs a :cpp:func:`UnaryUnion` which dissolves boundaries of all touching triangle patches and thus yields a slimmer dataset which often suffices for basic GIS usage. Be aware that this dissolving step increases processing time significantly.
+      Layer *RoadSignal* will be dissolved to a simple :cpp:class:`OGRPoint`.
+
+Examples
+--------
+
+- Translate OpenDRIVE road *ReferenceLine* elements (``<planView>``) to :ref:`Shapefile <vector.shapefile>` using :program:`ogr2ogr`. The desired :ref:`layer type <layer types>` which is to be extracted from the dataset is specified as the last parameter of the function call. 
+
+  ::
+
+    ogr2ogr -f "ESRI Shapefile" CulDeSac.shp CulDeSac.xodr ReferenceLine
+
+- Convert the whole OpenDRIVE dataset with all its different layers into a :ref:`GeoPackage <vector.gpkg>` using:
+
+  ::
+
+    ogr2ogr -f "GPKG" CulDeSac.gpkg CulDeSac.xodr
+
+- Convert the whole OpenDRIVE dataset with custom parameters :oo:`EPS` and :oo:`DISSOLVE_TIN` into a :ref:`GeoPackage <vector.gpkg>`:
+
+  ::
+
+    ogr2ogr -f "GPKG" CulDeSac.gpkg CulDeSac.xodr -oo EPS=0.9 -oo DISSOLVE_TIN=YES
+
+Convenient usage through docker image 
+-------------------------------------
+
+To use the XODR driver inside a docker image, first build the image from the corresponding docker image directory 
+    
+  ::
+
+    cd <gdal>/docker/ubuntu-small/
+    docker build -t gdal/xodr -f DockerfileXODR .
+
+For general usage information refer to `GDAL Docker images <https://github.com/OSGeo/gdal/tree/master/docker#usage>`__. Usage examples:
+
+- Use :program:`ogrinfo` to extract detailed information about a local `xodr` file by mounting your current working directory (`$PWD`) containing the file into the Docker container:
+  
+  ::
+
+    docker run --rm -v ${PWD}:/home -it gdal/xodr ogrinfo /home/<file>.xodr
+
+- Use :program:`ogr2ogr` to convert a local `xodr` file into any other supported OGR output format. The result will be automatically available in your host machine's working directory which is mounted into the container: 
+  
+  ::
+
+    docker run --rm -v ${PWD}:/home -it gdal/xodr ogr2ogr -f "GPKG" /home/<file>.gpkg /home/<file>.xodr
+
+
+Alternatively, you can run a docker container that enables using the XODR driver in an isolated workspace from within the container 
+    
+    ::
+
+      docker run --name <container_name> -it gdal/xodr /bin/bash
+
+
+General building notes
+----------------------
+
+Building of the driver as plugin is tested to work on
```

Comment:
Regarding building documentation, a section for libopendrive (similar to the other 66 libraries...) should be added to doc/source/development/building_from_source.rst

Reply:
We will add a specific section for libOpenDRIVE to `doc/source/development/building_from_source.rst`.

Should we then completely avoid "General building notes" in the actual driver documentation and also add these more comprehensive hints to `building_from_source.rst`?